plugins/woocommerce/client/admin/client/payments-welcome/index.tsx

```diff
@@ -114,6 +113,7 @@ const ConnectAccountPage = () => {
 			const installAndActivateResponse = await installAndActivatePlugins(
 				[ 'woocommerce-payments' ].concat( pluginsToInstall )
 			);
+			// @ts-expect-error TODO react-18-upgrade: useDispatch does not return the correct type for installAndActivatePlugins.
```

Comment:
It looks like `useDispatch` does not return the correct type for dispatch functions. It shows a Generator type for return value instead of the actual return type so I ignore the type error for now. I think we need to fix in global scope.  cc @rjchow 

<img width="773" alt="Screenshot 2025-02-13 at 16 11 06" src="https://github.com/user-attachments/assets/e8d01cd6-9bad-408c-870d-b1fab8ead6b7" />

Reply:
Ah ok thanks for catching that, let me have a look at it